.github/workflows/build.yml

```diff
@@ -349,7 +338,6 @@ jobs:
         run: |
           state run deploy-updates
           state run deploy-installers
-          state run deploy-msi
```

Comment:
Any reason to keep this script in the as.yaml?

Reply:
I didn't want to make this PR about removing the whole mechanic altogether, just wanted to remove it from our CI builds.